src/BlockTypesController.php

```diff
@@ -234,7 +235,7 @@ protected function get_atomic_blocks() {
 			'product-button',
 			'product-image',
 			'product-price',
-			'product-rating',
+			'product-sale-badge',
```

Comment:
I had to remove this line, otherwise I was getting an error telling me `product-sale-badge` was registered twice:


`( ! ) Notice: WP_Block_Type_Registry::register was called <strong>incorrectly</strong>. Block type "woocommerce/product-sale-badge" is already registered. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.0.0.) in .../Local Sites/latest/app/public/wp-includes/functions.php on line 5768`

![imatge](https://user-images.githubusercontent.com/3616980/154042161-3f643284-11cd-4a71-8285-ce2fd60935c7.png)